OWML.Common/IModConsole.cs

```diff
@@ -1,8 +1,14 @@
-﻿namespace OWML.Common
+﻿using System;
+
+namespace OWML.Common
 {
     public interface IModConsole
     {
-        void WriteLine(string s);
-        void WriteLine(params object[] s);
+        [Obsolete]
+        void WriteLine(params object[] objects);
+        [Obsolete]
+        void WriteLine(string line);
+
```

Comment:
Actually, why don't we keep this (not obsolete) and default message type to Message or Info (and remove the magic for guessing message type)?

Reply:
If we do this we won't need to change that many WriteLines for OWML